src/openzeppelin/access/accesscontrol/IAccessControl.cairo

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Contracts for Cairo v0.2.1 (access/accesscontrol/IAccessControl.cairo)
+
+%lang starknet
+
+@contract_interface
+namespace IAccessControl:
+    func has_role(role: felt, user: felt) -> (has_role: felt):
```

Comment:
```suggestion
    func hasRole(role: felt, user: felt) -> (has_role: felt):
```
functions need to be cameCased since this interface (which registers ERC165's `IACCESSCONTROL_ID`) is the same as [Solidity's](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/IAccessControl.sol). we could also make the case for renaming `user` to `account` for the same reason. note that currently it's the library that calls this param `user`, which has no immediate effect on the contracts' public interface but we might want to fix that as well.

Reply:
thank you for the feedback. will update accordingly!

Reply:
hi @martriay code updated!